src/NHibernate.Test/NHSpecificTest/Logs/LogsFixture.cs

```diff
@@ -30,6 +33,33 @@ protected override string MappingsAssembly
 			get { return "NHibernate.Test"; }
 		}
 
+		protected override void Configure(Configuration configuration)
+		{
+			base.Configure(configuration);
+			configuration.SetProperty(Cfg.Environment.UseSecondLevelCache, "false");
```

Comment:
`TestCase` base class causes all entities to be cacheable in second level cache by default, and enables it. Quite tricky for a test which relies on emitted SQL.